src/TezosProvider.ts

```diff
@@ -0,0 +1,509 @@
+import axios from "axios";
```

Comment:
I doubt you actually need axios. you could do with fetch easily. no need to bring unnecessary dependencies